### ISSUE TITLE
kb_{project}_edge is missing after update from 0.10.5 to 0.10.6

### ISSUE THREAD

**Issue by jpsalter** [kalabox/kalabox#781]
1. Updated to 0.10.6
2. Ran kbox update
3. Restarted computer
4. kbox start throws error for existing project
5. kbox rebuild throws error for existing project

Not sure how to proceed. Should I start over and rebuild the project from Pantheon again?

```
Error starting container kb_institute-of-governmental-studies_edge.: The container "'kb_institute-of-governmental-studies_edge'" does not exist!
Error: The container "'kb_institute-of-governmental-studies_edge'" does not exist!
    at /usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:266:15
From previous event:
    at findContainerThrows (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:264:6)
    at Object.start (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:445:12)
From previous event:
    at /usr/local/bin/lib/engine.js.jx:554:29
From previous event:
    at Object.exports.start (/usr/local/bin/lib/engine.js.jx:553:4)
    at /usr/local/bin/lib/app.js.jx:705:19
From previous event:
    at startComponent (/usr/local/bin/lib/app.js.jx:704:4)
    at services.verify.then.then.then.then.map.concurrency (/usr/local/bin/lib/app.js.jx:755:12)
From previous event:
    at exports.start (/usr/local/bin/lib/app.js.jx:754:4)
    at /usr/local/bin/lib/app.js.jx:862:12
From previous event:
    at Object.exports.restart (/usr/local/bin/lib/app.js.jx:861:4)
    at Object.task.func (/usr/local/bin/plugins/kalabox-core/tasks/restart.js.jx:15:18)
    at Object.exports.run (/usr/local/bin/lib/core/tasks.js.jx:876:15)
    at processTask (/usr/local/bin/bin/kbox.js.jx:170:16)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:233:12)
    at process._tickCallback (node.js:803:1)
From previous event:
    at main (/usr/local/bin/bin/kbox.js.jx:232:4)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:241:1)
    at Module._compile (module.js:595:24)
    at readX (module.js:1488:3)
    at Object.Module._extensions..jx (module.js:1921:8)
    at Module.load (module.js:416:30)
    at Function.Module._load (module.js:380:8)
    at Module.require (module.js:451:15)
    at require (module.js:469:13)
    at Object.<anonymous> (_jx_source:17:1)
    at Module._compile (module.js:595:24)
    at Module.load (module.js:425:6)
    at Function.Module._load (module.js:380:8)
    at Function.Module.runMain (module.js:813:8)
    at startup (node.js:374:8)
    at node.js:1461:1
```

**Comment by pirog** [comment]
Oh weird. We didn't have an edge container in 0.10.5 so that might be why you are getting that. 

I would destroy the app and try recreating as a workaround for now.

**Comment by jpsalter** [comment]
1. ran kbox destroy 
2. rebooted computer
3. ran kbox create pantheon

```
Error dispatching event listeners [post-create-app]: Failed after 5 retries. {"max":5,"backoff":500}: Cannot call method 'split' of undefined
VError: Failed after 5 retries. {"max":5,"backoff":500}: Cannot call method 'split' of undefined
    at /usr/local/bin/lib/promise.js.jx:61:17
From previous event:
    at rec (/usr/local/bin/lib/promise.js.jx:47:13)
    at /usr/local/bin/lib/promise.js.jx:55:20
From previous event:
    at /usr/local/bin/lib/promise.js.jx:54:12
    at process._tickCallback (node.js:803:1)
```

**Comment by labboy0276** [comment]
This is the same as my issue here:
https://github.com/kalabox/kalabox/issues/780

On Thu, Dec 3, 2015 at 1:47 PM, jpsalter <notifications@github.com> wrote:

>
>    1. ran kbox destroy
>    2. rebooted computer
>    3. ran kbox create pantheon
>
> Error dispatching event listeners [post-create-app]: Failed after 5 retries. {"max":5,"backoff":500}: Cannot call method 'split' of undefined
> VError: Failed after 5 retries. {"max":5,"backoff":500}: Cannot call method 'split' of undefined
>     at /usr/local/bin/lib/promise.js.jx:61:17
> From previous event:
>     at rec (/usr/local/bin/lib/promise.js.jx:47:13)
>     at /usr/local/bin/lib/promise.js.jx:55:20
> From previous event:
>     at /usr/local/bin/lib/promise.js.jx:54:12
>     at process._tickCallback (node.js:803:1)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalabox/kalabox/issues/781#issuecomment-161744477>.
>



-- 
*John Ouellet*
*Sr Developer & Support Manager*
415-470-3076
www.kalamuna.com

Stay connected with Kalamuna on: Twitter
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Ftwitter.com%2Fkalamuna&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>
, Linkedin
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Fwww.linkedin.com%2Fcompany%2F2584606&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>,
and Facebook
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Fwww.facebook.com%2Fkalamunatech&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>

**Comment by jpsalter** [comment]
As a workaround for the failed download:
1. kbox restart (does not throw errors and gets the site running)
2. kbox pull (pulls the remaining missing files)

**Comment by pirog** [comment]
Awesome! Thanks for posting a workaround while we look at the underlying
issue!

On Thu, Dec 3, 2015 at 1:21 PM, jpsalter <notifications@github.com> wrote:

> As a workaround for the failed download:
> 1. kbox restart (does not throw errors and gets the site running)
> 2. kbox pull (pulls the remaining missing files)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalabox/kalabox/issues/781#issuecomment-161788325>.
>



-- 

Mike

-- 
Mike Pirog
Co-Founder, Products
http://kalamuna.com

Stay connected with Kalamuna on: Twitter
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Ftwitter.com%2Fkalamuna&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>
, Linkedin
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Fwww.linkedin.com%2Fcompany%2F2584606&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>,
and Facebook
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Fwww.facebook.com%2Fkalamunatech&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>

**Comment by pirog** [comment]
This should be resolved in the 0.11.0 release on Friday
